lib/dialects/mssql/query-generator.js

```diff
@@ -407,15 +409,15 @@ module.exports = (function() {
         , existingConstraints = [];
 
       for (key in attributes) {
-        attribute = attributes[key];
+        attribute = Utils.formatReferences(attributes[key]);
```

Comment:
Please move this in the if clause after [line 414] (https://github.com/sequelize/sequelize/pull/3747/files#diff-7326c670d3f83779ae65be6a1e992deaR415) so it's not called if not necessary.